components/esm/loadregn.cpp

```diff
@@ -103,7 +105,10 @@ namespace ESM
         esm.writeHNT("CNAM", mMapColor);
         for (std::vector<SoundRef>::const_iterator it = mSoundList.begin(); it != mSoundList.end(); ++it)
         {
-            esm.writeHNT<SoundRef>("SNAM", *it);
+            esm.startSubRecord("SNAM");
+            esm.writeFixedSizeString(it->mSound, 32);
+            esm.writeT(it->mChance);
+            esm.endRecord("NPCO");
```

Comment:
Why `SNAM` record ends as `NPC0`? endRecord has assert to check whether argument matches last record and it fails now in `esmtool clone` for Morrowind.esm.

Reply:
Copypaste mistakes do this to you.